js-exercises/freq-sort/freqSort.js

```diff
@@ -0,0 +1,15 @@
+function frequencySort(...args) {
+  if (args[0].length === 0)
+    throw "";
+
+  let store = {};
+  args[0].forEach(function (e, i, a) {
```

Comment:
Should not use args inside the program. This makes the programme less readable.
Rather you should have done something like this on line 2 itself:
`let charArray = args[0]`
and then should have used charArray instead,